daemon/config.cfg

```diff
@@ -12,3 +12,7 @@ level=DEBUG
 [RPCSERVICE]
 host=0.0.0.0
 port=50000
+
+[Spotify_Parameters]
+Client_ID=e9676baa69da445bbd9b691658628b69
+Client_Secret=20727079480e4093a14529aefd950ffa
```

Comment:
I have to use id and secret to get token. Can I putting these in my source code?